.github/workflows/stale.yml

```
@@ -0,0 +1,48 @@
+name: 'Stale Issue and PR Cleanup'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.1.0
+        id: stale
+        with:
+          days-before-stale: 30
+          days-before-close: 14
+          exempt-all-assignees: false
+          exempt-draft-pr: true
+          stale-issue-label: "Pending Closure"
+          stale-pr-label: '0 - Waiting on User'
+          only-pr-labels: '0 - Waiting on User'

```

Based on your last comment, if the intention is that only issues **AND** PR's that have been labeled with the `0 - Waiting on User` should be considered to be stale,

then this value should be:

```suggestion
          only-labels: '0 - Waiting on User'

```

See Readme for the stale action: https://github.com/actions/stale#only-labels